techniques/aria/ARIA14.html

```diff
@@ -1,63 +1,98 @@
+<!DOCTYPE html>
+<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
+
+  <head>
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8" />
+    <title>Using aria-label to provide an accessible name where a visible label cannot be used</title>
+    <link rel="stylesheet" type="text/css" href="../../css/editors.css" class="remove" />
+  </head>
+
+  <body>
+    <h1>Using aria-label to provide an accessible name where a visible label cannot be used</h1>
+    <section class="meta">
+      <p class="id">ID: ARIA14</p>
+      <p class="technology">Technology: aria</p>
+      <p class="type">Type: Technique</p>
+    </section>
+    <section id="applicability">
+      <h2>When to Use</h2>
       <p>Technologies that support <a href="https://www.w3.org/TR/wai-aria/">Accessible Rich Internet Applications (WAI-ARIA)</a>. </p>
-   </section><section id="description"><h2>Description</h2>
-      <p>For sighted users, the context and visual appearance of an element can provide sufficient cues to determine the purpose. An example is the 'X' often used in the top right corner of pop-up <code class="language-html">div</code>s (light boxes) to indicate the control for closing the div.  
-</p>
-      <p>In some situations, elements can be given the attribute <code class="language-html">aria-label</code> to provide an accessible name for situations when there is no visible label due to a chosen design approach or layout but the context and visual appearance of the control make its purpose clear.</p>
+    </section>
+    <section id="description">
+      <h2>Description</h2>
+      <p>For sighted users, the context and visual appearance of an element can provide sufficient cues to determine the purpose. An example is the “&times;” often used in the top-right corner of dialogs to indicate the control for closing the dialog. While it might be visually clear that the “&times;” button closes the dialog, users with assistive technologies rely on accessible names that clearly communicate the purpose of components, in this case “Close”.
+      </p>
+      <p>When no clear visible text label is available due to design decisions, the accessible name can be set by using the <code class="language-html">aria-label</code> attribute instead.</p>
+
+<p>The <code class="language-html">aria-label</code> attribute can also be used to provide an accessible name for custom controls. Custom controls can’t use techniques for native HTML controls, like using a <code class="language-html">&lt;label&gt;</code> element with the <code class="language-html">for</code> attribute.</p>
```

Comment:
> Custom controls can’t use techniques for native HTML controls, like using a <code class="language-html">&lt;label&gt;</code> element with the <code class="language-html">for</code> attribute.

This isn't accurate either - you can absolutely use `<label>` elements as part of implementing custom controls. A `<label>` is allowed to label anything that the HTML spec considers ["labelable element"](https://html.spec.whatwg.org/multipage/forms.html#category-label), which includes, for example, [form-associated custom elements](https://html.spec.whatwg.org/multipage/custom-elements.html#form-associated-custom-element)

Reply:
not all though ... "custom controls" in the sense of, say, a mere `<div>` with a `role` etc can't be labelled with a `<label>`. so yeah, this needs to be mitigated/explained

Reply:
It's tough to talk about use of aria-label without ensuring considerations around role are also mentioned. Eric suggested more specific examples to support screen readers. A more thorough example -- maybe a working example? -- would be better.